components/team_members_dropdown/team_members_dropdown.tsx

```diff
@@ -48,7 +60,7 @@ export default class TeamMembersDropdown extends React.PureComponent {
         };
     }
 
-    handleMakeMember = async () => {
+    public handleMakeMember = async () => {
```

Comment:
If we're specifying visibility on these methods, they should probably be private